protecode/scanning.py

```diff
@@ -190,97 +169,86 @@ def iter_vulnerabilities_with_assessments(
 
     def scan_requests(
```

Comment:
As this function does only yield one scan request by now, let us rename it to `scan_request` and don't use a generator anymore.